components/faq/faq-board.vue

```diff
@@ -5,7 +5,7 @@
     </MCTitle>
     <hr>
     <div v-for="(faq,key) in faqs" :key="`faq-${key}`">
-      <div class="card-body" style="padding:1rem">
+      <div class="card-body" style="padding: 0 1rem">
```

Comment:
Use Bootstrap "px-1" for left and right padding.

Reply:
Updated to px-4 py-0.